doc/migrating-from-0.x.md

```diff
@@ -38,7 +38,26 @@ Replaced by `.formatMessage( path [, variables ] )`. The new API is quite differ
 
 ## Globalize.parseInt/parseFloat
 
-Replaced by `.parseNumber( value [, options] )`.
+Replaced by `.parseNumber( value [, options] )`.  `parseNumber` is an alias for [`.numberParser( [options] )( value )`](https://github.com/jquery/globalize/blob/master/doc/api/number/number-parser.md).  So where you might have previously executed:
+
+```js
+Globalize("en").parseFloat("123,456.789")
```

Comment:
For style consistency, let's add spaces, e.g., `Globalize( "en" ).parseFloat( "123,456.789" )`.